IAV-X: Load components common between projects from component library

lauriii opened the pull request:
See https://github.com/LaakarikeskusAava/react-component-library as well

juhog commented on package.json:
```diff
@@ -71,6 +71,7 @@
   "dependencies": {
     "axios": "^0.15.3",
     "babel-core": "^6.17.0",
+    "aava-react-component-library": "^0.1.1",
```
Isn't the github address of this package declared anywhere?

https://github.com/LaakarikeskusAava/react-component-library

lauriii replied:
nope, the package is available in the npm package manager

lauriii commented:
Following files has to be updated upstream:
```
Deleted merge conflict for 'src/components/MobileTable/MobileTable.js':
  {local}: deleted
  {remote}: modified file
Use (m)odified or (d)eleted file, or (a)bort? d

Deleted merge conflict for 'src/components/MobileTable/MobileTable.scss':
  {local}: deleted
  {remote}: modified file
Use (m)odified or (d)eleted file, or (a)bort? d

Deleted merge conflict for 'src/components/MobileTable/MobileTableSingleCol.js':
  {local}: deleted
  {remote}: modified file
Use (m)odified or (d)eleted file, or (a)bort? d

Deleted merge conflict for 'src/components/Throbber/Throbber.scss':
  {local}: deleted
  {remote}: modified file
Use (m)odified or (d)eleted file, or (a)bort? d

```

kallevu commented:
Created another PR for aava-react-component-library https://github.com/LaakarikeskusAava/react-component-library/pull/11

juhog commented:
Should we close this PR without merging?
